### PR TITLE
Added wrapper for position_camera_to_world

### PR DESCRIPTION
**Pull request by esteldunedain**

### CODE REVIEW

File: src/client/intercept/client/sqf/uncategorized.cpp [hunk 1]
```diff
@@ -6199,6 +6199,11 @@ namespace intercept {
             return __helpers::get_pos_obj(__sqf::unary__position__object__ret__array, obj_);
         }
 
+        vector3 position_camera_to_world(const vector3 & pos_)
+        {
```
Comment (jonpas) [inline src/client/intercept/client/sqf/uncategorized.cpp]: [Nope](https://github.com/NouberNou/intercept/wiki/Coding-Standards#control-braces)

Comment (esteldunedain) [inline src/client/intercept/client/sqf/uncategorized.cpp]: Did you see the functions around this one?

Comment (jonpas) [inline src/client/intercept/client/sqf/uncategorized.cpp]: Yes, they should be changed as well, they are the auto-generated ones.

**Comment by esteldunedain** [comment]
Also changed the opening brace of all functions to bring them to comply the code convention

**Comment by jonpas** [comment]
Nice :+1:

**Pull request merged**